src/lib/chat/matrix/chat-message.ts

```diff
@@ -179,7 +179,7 @@ function getRoomReactionAdminData(content, sender) {
 }
 
 function getRoomMemberAdminData(content, targetUserId, previousContent?) {
-  if (previousContent?.avatar_url !== content.avatar_url) {
+  if (previousContent?.avatar_url && previousContent?.avatar_url !== content.avatar_url) {
```

Comment:
why not add an empty check for `content.avatar_url` also? Well, i guess you can't do that from our UI 🤔